hvac/api/system_backend/quota.py

```diff
@@ -0,0 +1,112 @@
+from numpy import block
```

Comment:
I don't see this used anywhere and I don't think we have a dependency on numpy right now, was this auto-added by accident?

Reply:
Yes, that was an accident! Thank you